rust/stackable-cockpit/src/xfer/cache.rs

```diff
@@ -195,9 +260,27 @@ impl CacheSettings {
         match &self.backend {
             CacheBackend::Disk { base_path } => {
                 fs::create_dir_all(base_path).await.context(CacheIoSnafu)?;
-                Ok(Cache::new(self.backend, self.max_age))
+                let cache_auto_purge_filepath = base_path.join(CACHE_LAST_AUTO_PURGE_FILEPATH);
+
+                // Only create file if not already present
+                if !fs::try_exists(&cache_auto_purge_filepath)
+                    .await
+                    .context(CacheIoSnafu)?
+                {
+                    write_cache_auto_purge_file(cache_auto_purge_filepath).await?;
+                }
```

Comment:
Isn't it enough for `auto_purge()` to maintain this?

Reply:
`auto_purge()` now maintains this file since 2eaaa8b